opentelemetry-api/tests/util/test__importlib_metadata.py

```diff
@@ -24,11 +30,51 @@ def test_entry_points(self):
         self.assertIsInstance(
             next(
                 iter(
-                    entry_points(
+                    importlib_metadata_entry_points(
                         group="opentelemetry_meter_provider",
                         name="default_meter_provider",
                     )
                 )
             ).load()(),
             MeterProvider,
         )
+
+    def test_uniform_behavior(self):
+        """
+        Test that entry_points behaves the same regardless of the Python
+        version.
+        """
+
+        entry_points = importlib_metadata_entry_points()
+        self.assertIsInstance(entry_points, Dict)
+
+        entry_points = entry_points["opentelemetry_propagator"]
+        self.assertIsInstance(entry_points, Tuple)
+        self.assertIsInstance(entry_points[0], EntryPoint)
+
+        entry_points = importlib_metadata_entry_points(
+            group="opentelemetry_propagator"
+        )
+        self.assertIsInstance(entry_points, Tuple)
+        self.assertIsInstance(entry_points[0], EntryPoint)
+
+        entry_points = importlib_metadata_entry_points(
+            group="opentelemetry_propagator", name="baggage"
+        )
+        self.assertIsInstance(entry_points, Tuple)
+        self.assertIsInstance(entry_points[0], EntryPoint)
```

Comment:
Should have an assertion here of `group` and `name` values equals the `EntryPoint` values.

Reply:
Fixed